lib/server.js

```diff
@@ -59,6 +60,9 @@ function serverFactory(core, messageHandler, Logger) {
             logger.info('proxyDHCP server is listening on %s:%s'.format(self.address, self.inPort));
             self.server.setBroadcast(true);
         });
+
+        var DHCPPoller = new IscDhcpLeasePoller({}, {});
+        DHCPPoller.run();
```

Comment:
The ARP poller can be disabled by setting `serviceGraph` to false in it's graph definition.  Since the DHCP poller is not longer a graph it probably will need to be defined in the config.json.